src/test/resources/uk.gov.justice.laa.crime.cfecrime/features/integration/InitialMeansTest.feature

```diff
@@ -0,0 +1,18 @@
+@InitialMeansTest #@SmokeTest
+Feature: InitialMeansTest
+  @ValidMeansTest
+  Scenario: Get the InitialMeansTest calculation
+      Given the following input data and I call CMA
+        | InitAssessmentResult | FullAssessmentResult | MagistrateCourt | CaseType     | FullAssessmentPossible |
```

Comment:
IMHO it would be better if we split up the stub and the actual input data. That way when proper CMA comes along we just delete the stub line. If that's too hard, I think we should at least have them separate i.e. Result and FullAssessmentPossible collected with the actual inputs. We will also need more inputs, so copying the civil pattern would seem like a good idea

Reply:
I tried that, however the data did not seem to come through and did not want to spend too much time investigating.

Reply:
something like: Given I am doing a "<INITIAL>" means test with type "<CASETYPE>" and mag court "<COURT>"
And "CMA will return me"
<data table>
When I make a request to cfe-crime
Then The "<InitialMeamsTestResult>" is "<INELIGIBLE>"